src/templates/gmr-results.njk

```diff
@@ -0,0 +1,63 @@
+{% extends 'layout.njk' %}
```

Comment:
rename this file to `gmr-search-result.njk` to align with the existing `search-result.njk`